ui/src/app/pages/admin/user/admin-user-profile.spec.tsx

```diff
@@ -626,11 +626,11 @@ describe('AdminUserProfile', () => {
       },
     });
 
-    const excludedModules = [
+    const excludedModules: Array<AccessModule> = [
```

Comment:
If this array is not typed, it seems to assume the union of these string values rather than an array of this enum type.